### PR TITLE
Fix native aot test failures

### LINKED ISSUE #75699: [iOS] XslCompiledTransform tests failing on devices
https://github.com/dotnet/runtime/pull/75499 did some house cleaning and did not preserve skipping the entire Xsl suite.  XslCompiledTransform is not aot friendly and will result in messages like:

Rolling build [link](https://dev.azure.com/dnceng-public/public/_build/results?buildId=17272&view=ms.vss-test-web.build-test-results-tab&runId=343228&resultId=107474&paneView=debug)

```
System.ExecutionEngineException : Attempting to JIT compile method '(wrapper dynamic-method) void object:Execute (System.Xml.Xsl.Runtime.XmlQueryRuntime)' while running in aot-only mode. See https://docs.microsoft.com/xamarin/ios/internals/limitations for more information.


Stack trace
   at System.Delegate.CreateDelegate(Type type, Object firstArgument, MethodInfo method, Boolean throwOnBindFailure, Boolean allowClosed)
   at System.Reflection.Emit.DynamicMethod.CreateDelegate(Type delegateType)
   at System.Xml.Xsl.IlGen.XmlILModule.CreateDelegate(String name, Type typDelegate)
   at System.Xml.Xsl.XmlILGenerator.Generate(QilExpression query, TypeBuilder typeBldr)
   at System.Xml.Xsl.XslCompiledTransform.CompileQilToMsil(XsltSettings settings)
   at System.Xml.Xsl.XslCompiledTransform.LoadInternal(Object stylesheet, XsltSettings settings, XmlResolver stylesheetResolver, XmlResolver originalStylesheetResolver)
   at System.Xml.Xsl.XslCompiledTransform.Load(XmlReader stylesheet, XsltSettings settings, XmlResolver stylesheetResolver)
   at System.Xml.XslCompiledTransformApiTests.Errata4.TestXslTransform(Object param0, Object param1)
   at System.Reflection.MethodInvoker.InterpretedInvoke(Object obj, Span`1 args, BindingFlags invokeAttr)
```

### PR DESCRIPTION
**Pull request by jkotas**
Fixes #75699

### CODE REVIEW

**Comment by filipnavara** [comment]
You may want to cherry-pick https://github.com/dotnet/runtime/pull/76071/commits/f1c847253f7c3a4cd649389885993c5abd8f7504

File: src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertCreateCertificateContext.cs [hunk 1]
```diff
@@ -1,17 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```
Comment (jkotas) [inline src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertCreateCertificateContext.cs:1]: Delete some dead or duplicate files that I have run into along the way.

**Review by stephentoub — APPROVED** [review]

File: src/libraries/tests.proj [hunk 2]
```diff
@@ -489,14 +485,6 @@
...
 
-    <!-- These are failing in localized runs due to satellite assembly reflection in the test. Work fine otherwise. -->
-    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Xml\tests\XmlSchema\XmlSchemaValidatorApi\System.Xml.XmlSchema.XmlSchemaValidatorApi.Tests.csproj"
-                      Condition="'$(TargetOS)' == 'windows'"/>
-    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Xml\tests\Xslt\XslTransformApi\System.Xml.Xsl.XslTransformApi.Tests.csproj"
-                      Condition="'$(TargetOS)' == 'windows'"/>
-
-    <!--This test fails on Windows.10.Amd64.Server2022 -->
-    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.IO.FileSystem\tests\System.IO.FileSystem.Tests.csproj" />
     <!--Needs work to get these tests to pass -->
```
Comment (jkotas) [inline src/libraries/tests.proj:488]:
```suggestion
    <!--This test fails on Windows.10.Amd64.Server2022 -->
    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.IO.FileSystem\tests\System.IO.FileSystem.Tests.csproj" />
    <!--Needs work to get these tests to pass -->
```
Still failing

**Pull request merged**